lms/templates/dashboard/_dashboard_xseries_info.html

```diff
@@ -0,0 +1,35 @@
+<%page args="course_program_info, enrollment_mode" />
+<%!
+    from django.utils.translation import ugettext as _
+%>
+<%namespace name='static' file='../static_content.html'/>
+<div class="message message-status is-shown credit-message">
```

Comment:
I just caught this on stage. The class `credit-message` isn't appropriate here, since this isn't a credit message. Since this message is specific to XSeries, class should probably read `xseries-message`.